iOS/MSMapsModulesReferenceApp/Podfile

```diff
@@ -0,0 +1,12 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '9.0'
+
+target 'MSMapsModulesReferenceApp' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for MSMapsModulesReferenceApp
+  pod 'MicrosoftMapsSDK', '~> 1.1.4'
+  pod 'MSMapsModules', :git => 'https://github.com/microsoft/BingMapsNativeModules.git', :commit => '10c2c12c8fbd48041a768ad5036ae414ab4ac965'
```

Comment:
Move to local reference instead of via pod.

Reply:
I think that we need two apps. One, a test app which would use local reference and will allow for fast iteration for development. Second, a reference app which will contain minimum code to show how to make this code run. The reference app should depend on the pod. This will allow us to validate that everything works end to end.

Reply:
What is the difference between the test app and reference app other than where they import the pod from (local vs github)?

Reply:
The reference app is like a "Hello world" app. Simple example showing the users how to use your code. It does not showcase all uses of your library. Just lets the users get started. The test app can be more complex and contain some specific functionality for testing your library. If your test app is simple enough that you don't think it is worth to have both, you need to make it clear for the users about the two modes (local vs github) so that they know how to use it in their app.

Reply:
I see. Since the library only has one public method, it seems that the test app would overlap with the reference app a lot. For non-redundancy, I'll have one app and make a comment specifying the different places to get the library from.